challenges.yml

```
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
           },
           ```
       correct: a
+
+extending_css_transitions:
+  question: |
+    Imagine I have a simple component that can show a message when one of our
+    customers have bought too much pizza.
```

```suggestion
    customers has bought too much pizza.
```

That's tricky - it's because we are talking about "one" customer. I had to read it a few times :)

---

ha! Indeed, that got me too!